packages/plugin-core/src/web/extension.ts

```diff
@@ -95,3 +98,12 @@ async function setupTreeView(context: vscode.ExtensionContext) {
     });
   });
 }
+
+async function reportActivationTelemetry() {
+  const telemetryClient =
+    container.resolve<ITelemetryClient>("ITelemetryClient");
+
+  await telemetryClient.identify();
+  // TODO: Add workspace properties later.
+  await telemetryClient.track(VSCodeEvents.InitializeWorkspace);
```

Comment:
do we know that the user has initialized a workspace? if they have dendron installed and open up a regular code project, would this not also trigger?

Reply:
This will only report if they're inside a Dendron workspace, but it's more an artifact of some incorrect logic in the initialization flow.  I'll make a TODO note to fix the initialization bug.

Reply:
The thing we discussed a little while ago where `activate` doesn't get run again when a new folder gets opened in `vscode.dev`, which prevents us from initializing properly in that scenario.  So right now, Dendron web only really works in the `github.dev` hitting `.` flow.

Reply:
got it. can we add the TODO in the code?

Reply:
I added it in the latest commit (line 29 on this file)